sgv2-restapi/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# TODO: change to more modern JVM
```

Comment:
what stops us to do this now?

Reply:
nothing, @tatu-at-datastax will be doing this soon